src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,123 @@
+.cards {
+  margin: 0 auto;
+  padding: 50px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  column-gap: 48px;
+  row-gap: 46px;
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  cursor: pointer;
+  padding: 32px 16px 16px;
+  background: $white-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  transition: transform 0.3s ease-in-out;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $title-color;
+
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    color: $title-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $title-color;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding